### PR TITLE
DOC: Clean up instructions in ISSUE_TEMPLATE

### PR DESCRIPTION
**Pull request by gfyoung**
* Minor grammar fixes + clarification on where to paste the output of `pd.show_versions()` to ensure that it renders properly.
* Additional notes about ways to filter out duplicates and / or resolve issues by upgrading.

### CODE REVIEW

**Comment by gfyoung** [comment]
Not sure why Circle CI makes it less convenient to skip builds:

https://circleci.com/docs/1.0/skip-a-build/

**Comment by gfyoung** [comment]
I currently have added some points as "notes" to the issue template, but I was also considering make them checkboxes (similar to what we have in the PR template).  Which one do you think is better?

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -8,11 +8,18 @@
...
+if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.
+
 #### Expected Output
 
 #### Output of ``pd.show_versions()``
 
 <details>
-# Paste the output here pd.show_versions() here
+
+[paste the output of ``pd.show_versions()`` here below this line]
```
Comment (gfyoung) [inline .github/ISSUE_TEMPLATE.md]: A lot of people were just posting the output right below the "<details>" tag, causing it to not render properly.  This is an effort to guide people away from doing that.

File: .github/ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -8,11 +8,18 @@
 
 [this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
 
+**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before.
+Please check first before submitting so that we do not have to handle and close duplicates!
+
+**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check
+if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.
+
```
Comment (gfyoung) [inline .github/ISSUE_TEMPLATE.md]: I currently have added some points as "notes" to the issue template, but I was also considering make them check-boxes (similar to what we have in the PR template).  Which one do you think is better?

**Comment by jreback** [comment]
thanks!

File: .github/ISSUE_TEMPLATE.md (hunk 2 above)
Comment (jreback) [inline .github/ISSUE_TEMPLATE.md]: hah just actually read your comment again (after merging of course). this is fine.

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -8,11 +8,18 @@
 
 [this should explain **why** the current behaviour is a problem and why the expected output is a better solution.]
 
+**Note**: We receive a lot of issues on our GitHub tracker, so it is very possible that your issue has been posted before.
+Please check first before submitting so that we do not have to handle and close duplicates!
```
Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE.md]: you can put those notes in a comment, so if people don't remove it, it at least doesn't show up in the rendered issue 

it's like (from sklearn template)

```
<!--
If your issue is a usage question, submit it here instead:
- StackOverflow with the scikit-learn tag: http://stackoverflow.com/questions/tagged/scikit-learn
- Mailing List: https://mail.python.org/mailman/listinfo/scikit-learn
For more information, see User Questions: http://scikit-learn.org/stable/support.html#user-questions
-->
```

Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE.md]: actually, that content is maybe also useful to add :-)

File: .github/ISSUE_TEMPLATE.md [hunk 4]
```diff
@@ -8,11 +8,18 @@
...
+
+**Note**: Many problems can be resolved by simply upgrading `pandas` to the latest version. Before submitting, please check
+if that solution works for you. If possible, you may want to check if `master` addresses this issue, but that is not necessary.
```
Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE.md]: I personally would leave out this one. It is fully correct, but we have to be aware that the longer the text in here, the less people will read it. I would rather put it in our contributing docs and clearly link to that as a guide on how to submit an issue

**Review by jorisvandenbossche — COMMENTED** [review]
Some additional after-merge comments!

File: .github/ISSUE_TEMPLATE.md (hunk 4 above)
Comment (gfyoung) [inline .github/ISSUE_TEMPLATE.md]: @jorisvandenbossche : But following your logic, people are not very likely to read the contributing docs when it comes to submitting issues.  That's why I put that note here.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (gfyoung) [inline .github/ISSUE_TEMPLATE.md]: I think we've generally handled usage questions on the issue tracker from time-to-time.

Comment (jorisvandenbossche) [inline .github/ISSUE_TEMPLATE.md]: > I think we've generally handled usage questions on the issue tracker from time-to-time.

Yes, but that is not the purpose of the github issue tracker, and give only clutter (that said, I am very well aware it is not avoidable to a certain extent, but giving those pointers might help)

**Pull request merged**